app/templates/descriptor/confirm_resources.html

```diff
@@ -42,16 +42,12 @@ <h3 class="ui red block header">
                         <thead>
                             <tr>
                                 <th class="sorted ascending">Name</th>
-                                <th class="sorted ascending">{{ desc.name }}</th>
                             </tr>
                         </thead>
                         <tbody>
-                        {% for i in range(option_assocs|length) %}
+                        {% for oa in option_assocs %}
                             <tr>
-                                <td>{{ option_assocs[i].resource.name }}</td>
-                                <td class="descriptor value">
-                                    {{ f.render_form_field(form[option_assocs[i].resource.name])}}
-                                </td>
+                                <td><a href="{{ url_for('single_resource.edit', resource_id=oa.resource['id']) }}">{{ oa.resource.name }}</a></td></td>
```

Comment:
can we access the resource id directly from the option association? `oa.resource_id`?